boilerplate/app/i18n/ar.ts

```diff
@@ -32,6 +32,7 @@ const ar: TxType = {
     componentsTab: "عناصر",
     debugTab: "تصحيح",
     communityTab: "واصل اجتماعي",
+    podcastListTab: "البودكاست",
```

Comment:
Added Arabic translation. I don't know if it's correct. Just didn't want to bother Mazen on Friday evening.

Reply:
@mazenchami

Reply:
@kateinkim love it! some minor adjustments pushed